src/Tools/dotnet-watch/src/dotnet-watch.nuspec

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    $CommonMetadataElements$
+    <packageTypes>
+      <packageType name="DotnetTool" />
+    </packageTypes>
+  </metadata>
+  <files>
+    $CommonFileElements$
+    <file src="$PackageThirdPartyNoticesFile$" target=".\THIRD-PARTY-NOTICES.txt" />
+    <file src="DotnetToolSettings.xml" target="tools\$DefaultNetCoreTargetFramework$\any\DotnetToolSettings.xml" />
+    <file src="assets\DotNetWatch.targets" target="tools\$DefaultNetCoreTargetFramework$\any\assets\DotNetWatch.targets" />
+    <file src="$OutputPath$dotnet-watch.dll" target="tools\$DefaultNetCoreTargetFramework$\any\dotnet-watch.dll" />
+    <file src="$OutputPath$dotnet-watch.deps.json" target="tools\$DefaultNetCoreTargetFramework$\any\dotnet-watch.deps.json" />
+    <file src="$OutputPath$dotnet-watch.runtimeconfig.json" target="tools\$DefaultNetCoreTargetFramework$\any\dotnet-watch.runtimeconfig.json" />
+    <file src="$OutputPath$dotnet-watch.pdb" target="tools\$DefaultNetCoreTargetFramework$\any\dotnet-watch.pdb" />
+    <file src="$BrowserRefreshOutputPath$Microsoft.AspNetCore.Watch.BrowserRefresh.dll" target="tools\$DefaultNetCoreTargetFramework$\any\middleware\Microsoft.AspNetCore.Watch.BrowserRefresh.dll" />
+  </files>
+</package>
```

Comment:
I don't think this file will be used. Suggest removing the `PopulateNuspec` target and adding a `$(NuspecFile)` setting in the project file if you want this instead of the auto-generated file.

Reply:
You're right. This can be removed.